src/components/Slider/Slider.tsx

```diff
@@ -30,46 +31,26 @@ export const Slider = React.forwardRef(function Slider(
         errorMessage,
         validationState,
         disabled = false,
-        debounceDelay = 0,
         onBlur,
         onUpdate,
         onUpdateComplete,
         onFocus,
         autoFocus = false,
         tabIndex,
         className,
+        style,
         qa,
         apiRef,
         'aria-label': ariaLabelForHandle,
         'aria-labelledby': ariaLabelledByForHandle,
+        name,
+        form,
+        id,
```

Comment:
Are these props(`name`, `id`) fit to case with range slider, when value is [number, number]?

Reply:
Yes. `id` goes to container. And several inputs with the same name are valid (see tests).